DeathNotes.cs

```diff
@@ -381,9 +381,19 @@ private string GetEntityName(BaseEntity entity, CombatEntityType combatEntityTyp
                 case CombatEntityType.Helicopter:
                     return "Helicopter";
 
+                case CombatEntityType.Other:
+                    return "Other";
+
+                case CombatEntityType.None:
+                    return "None";
+
                 case CombatEntityType.Scientist:
                 case CombatEntityType.Murderer:
                 case CombatEntityType.Scarecrow:
+
+                    if(entity.ShortPrefabName == "heavyscientist")
+                        return "Heavy Scientist";
+

```

Comment:
This won't be necessary here, as I've added the Heavy Scientist to the Enemy Prefabs dictionary.
For that to become effective, I will have to move that statement from below above this switch-statement, but I will do that after your changes are through.